about.html

```diff
@@ -0,0 +1,110 @@
+
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <!--Google Fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet">
+    <script src="https://kit.fontawesome.com/681ec73d24.js" crossorigin="anonymous"></script>
+    <title>Oliver's Capstone Project</title>
+  </head>
+  <body>
+    <!--Menu-->
+    <header>
+      <nav class="language-navbar">
+        <ul class="navMenu">
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li class="lang-nav-item"><a class="top-link">English</a></li>
+          <li class="lang-nav-item"><a class="top-link">My Page</a></li>
+          <li class="lang-nav-item"><a class="top-link">Logout</a></li>
+        </ul>  
+      </nav>
+      <nav class="sticky">
+          <a href="#">
+            <img class="logo" src="./images/logotype.png" alt="logotype company">
+          </a>
+          <ul class="navbar menu">
+  
+              <li><a href="index.html" class="menu-i">Home</a></li>
+              <li><a href="#services" class="menu-i">Our Services</a></li>
+              <li><a href="#contact" class="menu-i">Contact Us</a></li>
+              <li><button type="button" class="sign-up btn"> Sign Up</button></li>
+          </ul>
+          <div class="menu-btn">
+              <i class="fas fa-bars"></i>
+          </div>
```

Comment:
**Suggested Changes**
- [ ] The mobile hamburger should be on the left side. Please let us move it.
- [ ] It's background is transparent or the same color/pattern as the top section background. Currently yours is white background.